_component_design/loading-indicators.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Loading indicators
+parent: Processing
+layout: component
+category: Components
+usage: >
+  Loading indicators tell the user that their system is actively retrieving data.
+preview-image: preview-images/loading.svg
+resource: true
+status: unknown
+last-modified: 2017-10-24
+---
+
+{% include toc.html %}
+
+## Introduction
+{: .hxSectionTitle}
```

Comment:
@nimbinatus Is currently building a plugin that will handle the adding of these classes without the need to add these via the kramdown method. Net:net remove all instances of `{: .hxSectionTitle}` and `{: .hxSubSectionTitle}` from this doc. Only noting this once in the review.